docs/conf.py

```diff
@@ -215,15 +215,24 @@
 
 # -- Options for the Sphinx gallery -------------------------------------------
 
-extensions += ["sphinx_gallery.gen_gallery"]
-
-sphinx_gallery_conf = {
-    'filename_pattern': '^((?!skip_).)*$', # execute all examples except those that start with "skip_"
-    'examples_dirs': '..{}examples'.format(os.sep), # path to the examples scripts
-    'gallery_dirs': 'auto_examples', # path to save gallery generated examples
-    'reference_url': {
-        'astropy': None,
-        'matplotlib': 'http://matplotlib.org/',
-        'numpy': 'http://docs.scipy.org/doc/numpy/',
+try:
+    import sphinx_gallery
+    extensions += ["sphinx_gallery.gen_gallery"]
+
+    sphinx_gallery_conf = {
+        'mod_example_dir': 'generated/modules', # path to store the module using example template
+        'filename_pattern': '^((?!skip_).)*$', # execute all examples except those that start with "skip_"
+        'examples_dirs': '..{}examples'.format(os.sep), # path to the examples scripts
+        'gallery_dirs': 'generated/examples', # path to save gallery generated examples
```

Comment:
@bsipocz - I thought the same at first, but decided not because that yields the final HTML urls as "docs.astropy.org/en/stable/_build/generated/whatever.html" which is rather ugly and reveals "implementation details" in the final product.  

I *could* also change it just `gallery` (or better yet `examples`, because I think "gallery" is a less-clear term for this... gallery of what?), but that then means that the sphinx command has to include `gallery`/`example`.  I think it's better to be consistent and put all generated things in `generated`, so that poor users who don't know sphinx very well can tell that that's where the generated things show up.  (One could imagine also moving the API to `generated/api`... But my thinking is that there are already lots of things on the web using the `docs.astropy.org/en/stable/api/...` URL, so it's better to maintain backwards compatibility)